pcm-player.js

```diff
@@ -23,7 +19,10 @@ PCMPlayer.prototype.init = function(options, onendedCallback) {
     this.typedArray = this.getTypedArray();
     this.onendedCallback = onendedCallback;
     this.feedCounter = 0;
-    this.createContext();
+}
+
+PCMPlayer.prototype.init = function() {
```

Comment:
It seems to me, it's not that critical.

The player's `init()` method is not supposed to be called manually.
It's either called by the [incomingMessage](https://github.com/zelloptt/zello-channel-api/blob/8089e73fcaf1097164bb42cdf5dfa6e04d9770ee/sdks/js/src/classes/incomingMessage.js#L150) or the [sdk](https://github.com/zelloptt/zello-channel-api/blob/8089e73fcaf1097164bb42cdf5dfa6e04d9770ee/sdks/js/src/classes/sdk.js#L134).

However if somebody uses the SDK player in his own implementation of the player - it will break indeed.

Reply:
Increased the version in package.json here (probably it doesn't affect at anything) and in the SDK.
Removed error catching, so it can be caught later by the SDK.